.dockerignore

```diff
@@ -42,9 +36,9 @@ lib64/
 parts/
 sdist/
 var/
-*.egg-info/
+# *.egg-info/
 .installed.cfg
-*.egg
+# *.egg
```

Comment:
It might? In development it will only work if you have `pip install -e .` or installed it. In the docker, it's building SMRF then installing it in `site-packages` so all that is contained in the package. Then you can check out `smrf.__version__` to get the version number.